src/background.ts

```diff
@@ -909,6 +909,7 @@ async function createWindow() {
       preload: path.join(__dirname, "preload.js"),
       nodeIntegration: true,
       contextIsolation: true,
+      sandbox: false,
```

Comment:
> sandbox: falseを指定しています。外しても問題ないならお知らせください。

こちら、わからないというのが正直なところです。
検証すれば良さそうですが、まあ付いてても良いのではともちょっと思います。
コメントでできれば外す、みたいなのをつけておくのはどうでしょう。

```suggestion
      sandbox: false, // TODO: 外しても問題ないか検証して外す
```

Reply:
ただ、`sandbox: false`をつけていないコミットのCIがコケているので多少の改修は必要かもしれませんね

Reply:
なぜかpackage-lockのhash化でコケてる気がしますね･･･